lib/matplotlib/contour.py

```diff
@@ -289,6 +289,7 @@ def get_label_width(self, lev, fmt, fsize):
 
         return lw
 
+    @cbook.deprecated("2.2")
```

Comment:
This seems unrelated.  Can you remove it or add an API changes doc?

Reply:
moved to its own PR